src/middlewares/natsToRestBridge.ts

```diff
@@ -10,27 +10,34 @@ const jsonCodec = JSONCodec();
 export default function restToNatsBridgeFactory(
     natsConnection: NatsConnection
 ) {
+    const hdr = headers();
+    let headerContent;
+
     return async function restToNatsBridge(
         req: Request,
         res: Response,
-        next: NextFunction
+        next: NextFunction,
     ): Promise<void> {
         const { method, path, body, query } = req;
+        headerContent = JSON.parse(JSON.stringify(req.headers));
+        if (headerContent.correlationid === undefined) {
```

Comment:
no need to assign a correlationId to the headerContent before assigning the value to `hdr`